.travis.yml

```diff
@@ -1,3 +1,8 @@
+dist: trusty
+
+addons:
+  chrome: stable
```

Comment:
Is the dist and addon change needed?  It looks like https://github.com/code-dot-org/craft/blob/master/.travis.yml is running ChromeHeadless without any added dependencies.

Reply:
Confirmed, it appears we don't need this: https://github.com/code-dot-org/dance-party/compare/use_karma_for_integration_tests...karma-next.